### PR TITLE
Arts new frontend design

### LINKED ISSUE #81: Arts#new frontend design
Create a html template for the Arts#new page and design it with scss

I reeeeealy like the idea behind Airbnb's listing form. If someone wanna tackle this feature it would probably inspire you a lot to connecdt to airbnb and look at how their form is. It's a multipage thing where each page ask one or two questions. Doesn't look like a form at all. It also has some videos and animations. It's brillant.

### PR DESCRIPTION
**Pull request by Glod33**
Working form, needs a set categories

### CODE REVIEW

File: app/controllers/arts_controller.rb [hunk 1]
```diff
@@ -15,13 +15,13 @@ def show
 
   def new
     @art = Art.new
+    @categories = Category.all
```
Comment (wJoenn) [inline app/controllers/arts_controller.rb:18]: How do you do your select if you remove the @categories ?

File: app/views/arts/new.html.erb [hunk 2]
```diff
@@ -1,2 +1,27 @@
...
+    <%= f.file_field :photos %>
+    <%= f.submit %>
+  <% end %>
+</div>
+
+still need:
+
+user guess that's already fixed
+
+ability to upload photos (working?)
```
Comment (wJoenn) [inline app/views/arts/new.html.erb:27]: Don't forget to remove those comments

File: app/views/arts/new.html.erb [hunk 3]
```diff
@@ -1,2 +1,27 @@
-<h1>Arts#new</h1>
-<p>Find me in app/views/arts/new.html.erb</p>
+<div class="form-container">
+  <%= form_with(model: @art, method: :post) do |f| %>
```
Comment (wJoenn) [inline app/views/arts/new.html.erb:2]: You don't need the parentheses nor the `method: :post`

**Review by wJoenn — CHANGES_REQUESTED** [review]
This is very much a work in progress.
I don't think there's really a need to push this to master if we end up replacing everything when doing the final product, which I'm nearly sure we will.

Do you wanna try working on that for real ? It's probably gonna take a few days because it's gonna require either a lot of JS or a lot of routes and in both case a lot of html and css so this could be your last issue of the project considering we have 3 days left

**Comment by MarianaCarlota** [comment]
Can we see a print screen? :)

**Review by MarianaCarlota — APPROVED** [review]

**Pull request closed without merging**